videocall-deepar/src/main/java/com/voximplant/demos/kotlin/videocall_deepar/stories/main/MainViewModel.kt

```diff
@@ -13,6 +16,9 @@ class MainViewModel : BaseViewModel(), AuthServiceListener {
     val moveToCall = MutableLiveData<Unit>()
     val moveToLogin = MutableLiveData<Unit>()
 
+    val localVideoPresetEnabled: LiveData<Boolean>
+        get() = voximplantCallManager.localVideoPresetEnabled
+
```

Comment:
I don't think that we should save the UI setting in the call manager layer.

It should be kept on the view model. When a call starts, it is better to pass this value to `VoximplantCallManager.createCall` 

the same needs to be refactored for call answer functionality

Reply:
Changes applied